docs/skopeo-sync.1.md

```diff
@@ -0,0 +1,80 @@
+% skopeo-sync(1)
+
+## NAME
+skopeo\-sync - Copy images from one or more repositories to a user specified destination.
+
+## SYNOPSIS
+**skopeo sync** [**--source-yaml**] _source_ _destination_
+
+## DESCRIPTION
+Copy images from one or more repositories to a user specified destination.
+
+Useful to keep in sync with a local container registry mirror. It can also be used to populate registries running inside of air-gapped environments.
+
+_source_ can be:
+ - a repository hosted on a container registry (eg: docker://registry.example.com/busybox)
+ - a local directory (eg: dir:/media/usb/).
+ - (when **--source-yaml** is specified) a YAML file with a list of source images from different container registries.
+
+When the source location is a container repository and no tags are specified,  **skopeo sync** will copy all the tags associated to the source image.
+
+_destination_ can be either a container registry (eg: docker://my-registry.local.lan) or a local directory (eg: dir:/media/usb).
+
+When _destination_ is a local directory, one directory per 'image:tag' will be created.
+
+## OPTIONS
```

Comment:
Please make sure that the capitalization of the list is consistent. Some descriptions start with caps and some don't.